src/app/clients/ActionBar.tsx

```diff
@@ -183,6 +235,13 @@ const ActionBar: React.FC<Props> = ({ selected, data }) => {
                     Actions
                 </Button>
             </OuterDiv>
+            {modalError === null ? (
+                <></>
+            ) : (
+                <AlertBox>
+                    <Alert severity="error">{modalError}</Alert>
+                </AlertBox>
+            )}
```

Comment:
What about `{modalError === null && <AlertBox> ... </AlertBox>}`

Reply:
Also actually do we have to use the color from the `themes.tsx` - as currently it is probably using the `Alert` default color

Reply:
If we are using this alert thing for general table related components, and if we are to use our default theme coloring - then it might worth making it a component in table to be used across all pages with tables